test/logitnormal.jl

```diff
@@ -0,0 +1,70 @@
+# Tests on Univariate LogitNormal distributions
+
+using Distributions
+using Random, Test
+using StatsFuns
+
+
+function tmpF()
```

Comment:
is this needed?

Reply:
I removed the `tmpF()` where I stored interactive code.
The usage of `StatsFuns` is necessary for the logistic function used at line 21.